wpeverest/everest-forms
# Tweak - Tooltip information enhancements

### All Submissions:






### Changes proposed in this Pull Request:
Added tooltip in Individual Form settings. Also, added documentation link in tooltip so that user can get further more information about the feature by clicking on the Learn More link


### How to test the changes in this Pull Request:

1. Go to any individual form created.
2. Visit Settings->General and Email. Hover over the tooltip to see the feature information and learn more links where necessary
3. Visit Everest Forms->Settings->reCaptcha and hover over the tooltip

### Types of changes:




### Other information:




### Changelog entry

> Tweak -  Tooltip information enhancements
